src/platform/site-wide/sass/consolidated-patches.scss

```diff
@@ -30,3 +30,7 @@ footer.footer {
     color: $color-white;
   }
 }
+
+#login-root {
+  z-index: 999;
+}
```

Comment:
@RobAWilkinson @accbjt Is this a safe change for the slide-out sidebar?

Reply:
@bshyong @ncksllvn This should be okay.